src/svgGeometry.js

```diff
@@ -151,11 +151,35 @@ export const sqrtPath = function(
     return path;
 };
 
-export const path: {[string]: string} = {
-    // Two paths that cover gaps in built-up parentheses.
-    leftParenInner: `M291 0 H417 V300 H291 z`,
-    rightParenInner: `M457 0 H583 V300 H457 z`,
+export const innerPath = function(name: string, height: number ): string {
+    // The inner part of stretchy tall delimiters
+    switch (name) {
+        case "\u239c":
+            return `M291 0 H417 V${height} H291z M291 0 H417 V${height} H291z`;
```

Comment:
It looks like the same path is being repeated twice.  Am I misunderstanding what this SVG path is doing?

Reply:
> It looks like the same path is being repeated

Correct. See https://github.com/KaTeX/KaTeX/blob/master/src/stretchy.js#L96-L106